templates/terraform/examples/backend_service_global_internal.tf.erb

```diff
@@ -0,0 +1,100 @@
+resource "google_compute_global_forwarding_rule" "default" {
```

Comment:
Does it make sense for this file to be part of the examples? I can also make it a separate test file.

What is best practice on this sort of thing?

Reply:
Unfortunately it's a bit of a best-judgment thing right now. I'd say for this one since global forwarding rules need backend services but not the other way around, that this example belongs with global forwarding rule and not backend service. My inclination is that the docs page should have realistic-ish examples, but not too many of them. So since this reads as a pretty typical way to set up load balancing, I'd say putting it in the docs is perfectly reasonable, but adding 5 more examples with different tweaks of the configuration wouldn't be.

One thing to be aware of with beta fields are that the generated tests don't have the import step, so it's a good idea to verify those manually as well.

Also it looks like the GFR docs didn't get generated for some reason, not really sure what's going on there.